pulp_container/app/token_verification.py

```diff
@@ -24,17 +25,14 @@ def _decode_token(encoded_token):
         "audience": CONTENT_HOST,
     }
     with open(settings.PUBLIC_KEY_PATH, "rb") as public_key:
-        try:
-            decoded_token = jwt.decode(encoded_token, public_key.read(), **JWT_DECODER_CONFIG)
-        except jwt.exceptions.InvalidTokenError:
-            decoded_token = {"access": []}
+        decoded_token = jwt.decode(encoded_token, public_key.read(), **JWT_DECODER_CONFIG)
     return decoded_token
 
 
 def _access_scope(request):
     repository_path = request.resolver_match.kwargs.get("path", "")
 
-    if request.method in ["GET", "HEAD", "OPTIONS"]:
+    if request.method in SAFE_METHODS:
```

Comment:
Nice use of predefined constant from DRF.

Reply:
Yeah, i am still constantly learning about DRF. ;)